openmc/data/data.py

```diff
@@ -0,0 +1,385 @@
+                    line[100:106] + '.' + line[107:112])
+                _ATOMIC_MASS[name.lower()] = mass
+
+        # For isotopes found in some libraries that represent all natural
+        # isotopes of their element (e.g. C0), calculate the atomic mass as
+        # the sum of the atomic mass times the natural abudance of the isotopes
+        # that make up the element.
+        for element in ['C', 'Zn', 'Pt', 'Os', 'Tl']:
+            isotope_zero = element.lower() + '0'
+            _ATOMIC_MASS[isotope_zero] = 0.
+            for iso, abundance in NATURAL_ABUNDANCE.items():
+                if re.match(r'{}\d+'.format(element), iso):
+                    _ATOMIC_MASS[isotope_zero] += abundance * \
+                                                  _ATOMIC_MASS[iso.lower()]
+
+    # Get rid of metastable information
+    if '_' in isotope:
+        isotope = isotope[:isotope.find('_')]
+
+    return _ATOMIC_MASS[isotope.lower()]
+
+
+def atomic_weight(element):
+    """Return atomic weight of an element in atomic mass units.
+
+    Computes an average of the atomic mass of each of element's naturally
+    occurring isotopes weighted by their relative abundance.
+
+    Parameters
+    ----------
+    element : str
+        Name of element, e.g. 'H', 'U'
+
+    Returns
+    -------
+    float
+        Atomic weight of element in [amu]
+
+    """
+    weight = 0.
+    for nuclide, abundance in NATURAL_ABUNDANCE.items():
+        if re.match(r'{}\d+'.format(element), nuclide):
+            weight += atomic_mass(nuclide) * abundance
+    if weight > 0.:
+        return weight
+    else:
+        raise ValueError("No naturally-occurring isotopes for element '{}'."
+                         .format(element))
+
+
+def water_density(temperature, pressure=0.1013):
+    """Return the density of liquid water at a given temperature and pressure.
+
+    The density is calculated from a polynomial fit using equations and values
+    from the 2012 version of the IAPWS-IF97 formulation.  Only the equations
+    for region 1 are implemented here.  Region 1 is limited to liquid water
+    below 100 [MPa] with a temperature above 273.15 [K], below 623.15 [K], and
+    below saturation.
+
+    Reference: International Association for the Properties of Water and Steam,
+    "Revised Release on the IAPWS Industrial Formulation 1997 for the
+    Thermodynamic Properties of Water and Steam", IAPWS R7-97(2012).
+
+    Parameters
+    ----------
+    temperature : float
+        Water temperature in units of [K]
+    pressure : float
+        Water pressure in units of [MPa]
+
+    Returns
+    -------
+    float
+        Water density in units of [g/cm^3]
+
+    """
+
+    # Make sure the temperature and pressure are inside the min/max region 1
+    # bounds.  (Relax the 273.15 bound to 273 in case a user wants 0 deg C data
+    # but they only use 3 digits for their conversion to K.)
+    if pressure > 100.0:
```

Comment:
Should this be `>= 100.0`? The docstring above reads `Region 1 is limited to liquid water below 100 [MPa]` indicating that `100.0` is also out of the bounds.